src/app/client/src/app/modules/groups/components/add-member/add-member.component.ts

```diff
@@ -1,15 +1,44 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Output, EventEmitter } from '@angular/core';
+import { ResourceService, ToasterService } from '@sunbird/shared';
+import { GroupsService } from '../../services';
+import { ActivatedRoute } from '@angular/router';
 
 @Component({
   selector: 'app-add-member',
   templateUrl: './add-member.component.html',
   styleUrls: ['./add-member.component.scss']
 })
-export class AddMemberComponent implements OnInit {
+export class AddMemberComponent {
+  public memberCreationStep = 1;
+  public invalidUserid = false;
+  @Output() memberCreation = new EventEmitter<any>();
+  public userid = '';
+  constructor(public resourceService: ResourceService, public toasterService: ToasterService,  public groupsService: GroupsService,
+    private activatedRoute: ActivatedRoute) { }
 
-  constructor() { }
+  VerifyMemberUserId() {
+    if (this.userid) {
+      this.invalidUserid = false;
+      this.memberCreationStep ++;
+    } else {
+      this.invalidUserid = true;
+    }
+  }
 
-  ngOnInit() {
+  async addUserToGroup() {
+    const routeParams: any = this.activatedRoute.snapshot.params;
+    const groupid = routeParams.groupId;
+    const userid = this.userid;
+    try {
+      const groupMember = await this.groupsService.addMemberToGroup(userid, groupid);
+      if (groupMember) {
+        this.toasterService.success('Member added successfully');
```

Comment:
This message should be from resource bundles

Reply:
Resolved